snapshot-labs/sx-monorepo
# fix: enable delegation dashboard for offchain ERC-20 delegation

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/177

This PR will enable the delegation dashboard for offchain spaces using Compound Governor (ERC-20) delegation

### How to test

1. Go to an offchain which has setup Compound Governor delegation (http://localhost:8080/#/s:ens.eth/delegates) (should works by default on ens, gitcoin and uniswap DAO)
2. It should show the delegation dashboard for compound-governor delegation type (in addition to their "delegate-registry" delegation if any)
3. You should be able to delegate

### To-Do

- [ ] Waiting for manual database update to add delegation portal settings to spaces (ens, gitcoin and uniswap)
